### PR TITLE
add translate support to poll #408

### PR DESCRIPTION
**Pull request by Krishna-Singhal**

### CODE REVIEW

File: userge/plugins/utils/translate.py [hunk 1]
```diff
@@ -33,8 +33,13 @@
 async def translateme(message: Message):
     text = message.filtered_input_str
     flags = message.flags
-    if message.reply_to_message:
+    replied = message.reply_to_message
+    if replied:
         text = message.reply_to_message.text or message.reply_to_message.caption
+    elif replied.poll:
```
Comment (Jigarvarma2005) [inline userge/plugins/utils/translate.py]: Add this in `if replied:`, else it will not check this condition 🤔

**Comment by Jigarvarma2005** [comment]
```
if replied:
        if replied.poll:
            text = f'{replied.poll.question}\n'
            for i, option in enumerate(replied.poll.options, start=1):
                text += f'\n\t{i}.{option.get("text")}'
        else:
            text = message.reply_to_message.text or message.reply_to_message.caption
```

**Comment by Krishna-Singhal** [comment]
> ```
>     if replied:
>         if replied.poll:
>             text = f'{replied.poll.question}\n'
>             for i, option in enumerate(replied.poll.options, start=1):
>                 text += f'\n\t{i}.{option.get("text")}'
>         else:
>             text = message.reply_to_message.text or message.reply_to_message.caption
> ```

ah lol

**Review by Jigarvarma2005 — APPROVED** [review]

**Comment by rking32** [comment]
@Krishna-Singhal thanks for helping to close that issue :)

**Pull request merged**